src/main/java/org/fundacionjala/coding/ketty/Ean.java

```diff
@@ -0,0 +1,57 @@
+package org.fundacionjala.coding.ketty;
+
+/**
+ * verifica que un string  EAN.
+ */
+public class Ean {
+
+    /**
+     * El  CheckSum  valida si num es "EAN".
+     *
+     * @param num es la variable  String.
+     * @return return un valor boolean true es "EAN" y return false no es "EAN".
+     */
+    public boolean checkSum(String num) {
+        final int numeroMultiplicadorPar = 3;
+        int index = 1;
+        char numeroChar = ' ';
+        int suma = 0;
+        for (int i = 0; i < num.length() - 1; i++) {
+            numeroChar = num.charAt(i);
+            if (index % 2 == 0) {
+                suma += Character.getNumericValue(numeroChar) * numeroMultiplicadorPar;
+                index++;
+            } else {
+                suma += Character.getNumericValue(numeroChar) * 1;
+                index++;
+            }
+        }
+        System.out.println(suma);
+        int x = num.length() - 1;
+        char ni = num.charAt(x);
+        if (comprovacion(suma) == Character.getNumericValue(ni)) {
+            System.out.println("true");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)